deploy/helm/lunettes/templates/audit-apiserver/audit-apiserver-cm.yaml

```diff
@@ -1,8 +1,12 @@
 {{- if .Values.enableAuditApiserver }}
+{{- $policyVersion := default "v1beta1" -}}
+{{- if semverCompare ">=1.21-0" .Capabilities.KubeVersion.GitVersion -}}
```

Comment:
What version does "semverCompare" represent?
if it represent the version of kubernetes,  i can not understand the "semver" ?

Reply:
"SemverCompare" is a built-in function
![image](https://github.com/alipay/container-observability-service/assets/77275773/86e0ed05-8bb7-49ad-8986-6bbacf4a2d04)

Reply:
Has these code been tested?

Reply:
> Has these code been tested?

Yes, there are test result images above it